### DISCUSSION TITLE
.NET framework support in GitHub

### DISCUSSION THREAD

**Discussion by manushi129** [community/community#39313] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to build and pack the code using MSBuild command in GitHub action.
I'm using MSBuild -t: pack command for the same.
I'm facing the below error:
error MSB4057: The target "Pack" does not exist in the project.

**Comment by shutirtha-roy** [comment]
You can check this [url ](https://stackoverflow.com/questions/41021034/error-msb4057-missing-target-pack-when-building-netstandard-nuget-package)in stackoverflow.
